# This message is intended primarily for debugging and visualisation purposes

std_msgs/Header header
BlobDetection[] blobs
